examples/dataset/object_detection_3d/object_detection_3d/upload_results.py

```diff
@@ -292,16 +290,22 @@ def compute_metrics_with_difficulty(difficulty: int) -> pd.DataFrame:
                     unmatched_inference=unmatched_inference,
                     nInferences=len(record.raw_inferences_3d),
                     nValidObjects=sum(1 for ignore in ignored_gts_combined[i] if not ignore),
-                    nMatchedInferences=len(TP_2D),
-                    nMissedObjects=len(FN_2D),
-                    nMismatchedInferences=len(FP_2D),
+                    nMatchedInferences=len(TP_3D),
+                    nMissedObjects=len(FN_3D),
+                    nMismatchedInferences=len(FP_3D),
                     thresholds=current_optimal_thresholds,
-                    FP_2D=FP_2D,
                     FP_3D=FP_3D,
-                    TP_2D=TP_2D,
                     TP_3D=TP_3D,
-                    FN_2D=FN_2D,
                     FN_3D=FN_3D,
+                    images_with_inferences=[
+                        ImageAsset(
```

Comment:
A couple of things to consider
- Should we include (un)matched_inferences and unmatched_ground_truth on these images too? These would not be automatically hidden as they are at the top level and may clutter the image.
- This leads to one field on the dataset with just ground truths projected on all images and one field on the model result with both ground truths and model inferences projected on all images.

Reply:
Synced up on slack -- seems like the right approach for this longer term is to get the top level 3d bboxes projected into the 2d image assets (if they have a projection matrix). This looks to be non-trivial, so for the kitti example (for now), we'll have the left/right view visible only for model results (and include gts on those images).

Reply:
These changes^ are made now